builder/package.centos-7

```diff
@@ -0,0 +1,46 @@
+if [[ ! -d /tmp/output/centos-7 ]]; then
+  mkdir -p /tmp/output/centos-7
+fi
+
+fpm \
+  -s dir \
+  -t rpm \
+  -v 1 \
+  -n R-${R_VERSION} \
+  --vendor "R Project" \
+  --deb-priority "optional" \
+  --deb-field 'Bugs: https://github.com/rstudio/r-builds/issues' \
+  --url "http://www.r-project.org/" \
+  --description "GNU R statistical computation and graphics system" \
+  --maintainer "RStudio Inc https://github.com/rstudio/r-builds" \
+  --license "GPL-2" \
+  -p /tmp/output/centos-7/ \
+  -d bzip2-devel \
+  -d gcc \
+  -d gcc-c++ \
+  -d gcc-gfortran \
+  -d libcurl-devel \
+  -d libicu-devel \
+  -d libSM \
+  -d libtiff \
+  -d libXmu \
+  -d libXt \
+  -d make \
+  -d openblas-devel \
```

Comment:
There isn't a sensible way to work around this unfortunately. Ultimately the dependency is on those packages, rather than EPEL, so in situations where EPEL is not permitted users can still manually download those packages and install them first. I think this is as good as it's going to get, sadly.

Reply:
Aw ok, but that makes sense. But at least this is a whole lot better than not having an installer :)